src/main/java/org/fenixedu/treasury/domain/paypal/PayPal.java

```diff
@@ -180,8 +183,11 @@ private ForwardPaymentStatusBean prepareCheckout(ForwardPaymentRequest forwardPa
                 forwardPayment.setRedirectUrl(links);
             });
             DateTime requestReceiveDate = new DateTime();
-            String requestLog = new JSONObject(new Json().serialize(orderRequest)).toString(4);
-            String responseLog = new JSONObject(new Json().serialize(response.result())).toString(4);
+            JsonObject requestObject = GSON.toJsonTree(new Json().serialize(orderRequest)).getAsJsonObject();
```

Comment:
@anilmamede although we do not expect any suprises in this change to use GSON. Can you make sure we get this part validated? Thank you!